contracts/Manager.sol

```diff
@@ -200,7 +200,7 @@ contract Manager is Ownable {
             uint256 balance = IERC20(token).balanceOf(address(vault)); // unit: qRSV
 
             // Return false if this token is undercollateralized:
-            if (rsv.totalSupply().mul(weight) < balance.mul(scaleFactor)) {
+            if (rsv.totalSupply().mul(weight) > balance.mul(scaleFactor)) {
```

Comment:
This is an extremely important change. 

But I'm not sure why github is flagging it in the diff. I pushed the fix to `reviewed` over the weekend and if you check out the code in `reviewed` currently it does in fact show a `>` sign, which is correct.